packages/create-instance/create-component-stubs.js

```diff
@@ -80,25 +80,42 @@ function resolveOptions(component, _Vue) {
   return options
 }
 
-function getScopedSlotRenderFunctions(ctx: any): Array<string> {
-  // In Vue 2.6+ a new v-slot syntax was introduced
-  // scopedSlots are now saved in parent._vnode.data.scopedSlots
-  // We filter out the _normalized and $stable key
-  if (
-    ctx &&
-    ctx.$options &&
-    ctx.$options.parent &&
-    ctx.$options.parent._vnode &&
-    ctx.$options.parent._vnode.data &&
-    ctx.$options.parent._vnode.data.scopedSlots
-  ) {
-    const slotKeys: Array<string> = ctx.$options.parent._vnode.data.scopedSlots
-    return keys(slotKeys).filter(x => x !== '_normalized' && x !== '$stable')
-  }
-
-  return []
+function getAttributes(
+  functional: boolean,
+  functionalCtx: any,
+  componentCtx
+): Object {
+  return functional
+    ? {
+        ...functionalCtx.props,
+        ...functionalCtx.data.attrs,
+        class: createClassString(
+          functionalCtx.data.staticClass,
+          functionalCtx.data.class
+        )
+      }
+    : componentCtx.$props
 }
 
+// function getScopedSlotRenderFunctions(ctx: any): Array<string> {
+//   // In Vue 2.6+ a new v-slot syntax was introduced
+//   // scopedSlots are now saved in parent._vnode.data.scopedSlots
+//   // We filter out the _normalized and $stable key
+//   if (
+//     ctx &&
+//     ctx.$options &&
+//     ctx.$options.parent &&
+//     ctx.$options.parent._vnode &&
+//     ctx.$options.parent._vnode.data &&
+//     ctx.$options.parent._vnode.data.scopedSlots
+//   ) {
+//     const slotKeys: Array<string> = ctx.$options.parent._vnode.data.scopedSlots
+//     return keys(slotKeys).filter(x => x !== '_normalized' && x !== '$stable')
+//   }
+//
+//   return []
+// }
+
```

Comment:
So @lmiller1990 I'm mostly concerned about why no tests break when I comment this out